TACC/Core-Styles
# fix: c-card--frontera variant misc bugs

## Overview

Fix various bugs with `c-card--frontera` pattern and demo.

## Related

- noticed while testing https://github.com/TACC/tup-ui/pull/362

## Changes

- **added** style to undo base card padding
- **added** `.o-section` wrapper around demo markup
- **removed** load of redundant `supportStyles`

## Testing

1. Open http://localhost:3000/components/detail/c-card--frontera.
2. Hover over a card that is a link.
3. Verify outline on hover is fully visible.
4. Verify card has no excess padding e.g. outline is consistently offset around card content.
5. Verify `c-card--frontera` styles are not applied twice.
6. Verify pattern in demo looks like pattern on https://dev.fronteraweb.tacc.utexas.edu/2022/about/.\
    Exceptions:
    - outline color (which is site accent color)

## UI

| new | old |
| - | - |
| ![new](https://github.com/TACC/Core-Styles/assets/62723358/f1af5800-348e-4e06-ad46-5adfef89935c) | ![old](https://github.com/TACC/Core-Styles/assets/62723358/70c5ff5e-5f4f-45c0-8532-29a9f064e8ea) |
